Cargo.toml

```diff
@@ -3,18 +3,42 @@ authors = ["Marc Brinkmann <git@marcbrinkmann.de>"]
 description = "Linux SocketCAN library. Send and receive CAN frames via CAN bus on Linux."
 documentation = "https://docs.rs/socketcan"
 license = "MIT"
+edition = "2018"
 name = "socketcan"
 repository = "https://github.com/mbr/socketcan-rs"
 version = "2.0.0"
 
+# Features:
+#
+# "netlink" (default) - Whether to include CAN interface configuration 
+#        capabilities based on netlink kernel communications
+# "utils" - Build command-line utilities
+#
+
+[features]
+default = ["netlink"]
+netlink = ["neli"]
+vcan_tests = ["netlink"]
+utils = ["clap", "anyhow"]
+
 [dependencies]
+embedded-hal = "1.0.0-alpha.8"
```

Comment:
Their release appears to be a moving target. The `alpha.9` has breaking changes incompatible with this PR. If you want to keep this as-is, you would need to pin this exact version:
```
embedded-hal = "=1.0.0-alpha.8"
```